lib/gpx/geojson.rb

```diff
@@ -0,0 +1,222 @@
+#--
+# Copyright (c) 2017  Nick Knipe
```

Comment:
Of course! I was just tracking authorship as set forth in the other files in this gem -- license of course was the same.